wolfcrypt/src/sha.c

```diff
@@ -879,7 +879,11 @@ int wc_ShaFinal(wc_Sha* sha, byte* hash)
 
     XMEMCPY(hash, (byte *)&sha->digest[0], WC_SHA_DIGEST_SIZE);
 
-    ret = InitSha(sha); /* reset state */
+    {
```

Comment:
Please revert this code to the old version before Jim's PR. `(void)InitSha(sha); /* reset state */`

Reply:
:heavy_check_mark: